.huskyrc

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
It's ok to delete the .huskyrc configs since we want to have them align with what we're doing in the other repos 👍

Reply:
Ok, nice! Thanks!

Reply:
Ok, i was looking and when i pushed the other changes, it was not passing the tests on CircleCi, it was because its running yarn with lock files (I think thats because when you yarn install, that can take some time).
So i re-upload the yarn.lock again